addons/web/static/src/core/notebook/notebook.scss

```
@@ -1,18 +1,65 @@
+// Variables
+$o-notebook-horizontal-padding: 16px !default;
+$o-notebook-negative-margin: -$o-notebook-horizontal-padding !default;
+
+
+// SCSS
 .o_notebook {
-    &.horizontal {
-        flex-direction: column;
+    > .o_notebook_headers {
+        margin: 0px $o-notebook-negative-margin;
 
-        .nav.nav-tabs {
-            flex-flow: row nowrap;
+        > .nav {
+            padding: 0px $o-notebook-horizontal-padding;
```

nope. Use CSS variables
```suggestion
            padding: --var(--notebook-padding-x,  0px {$o-horizontal-padding});
```
By default it will fallback to the second value since `--notebook-padding-x` is normally not defined.

Then we can define `--notebook-padding-x` contextually (eg. inside form view or modals) .
Do the same for any property that we know will be overridden. Doing so in enterprise we'll have just something like this:

```scss
.o_notebook {
    --notebook-padding-x: 50px;
    --notebook-padding-y: 20px;
    --notebook-link-border-color: {$border-color};
    --notebook-link-active-border-color: {$primary};
}

.formView {
   [form view variables]
   
    --notebook-padding-x: 500px;
    --notebook-padding-y: 202px;
}
```